public/assets/styles/site.css

```diff
@@ -0,0 +1,195 @@
+/* Oh Well */
+
+:root {
+  --foreground: #FFFFFF;
+  --background: #212121;
+  --background-dark: #000000;
+  --dark-accent-0: #444444;
+  --dark-accent-1: #333333;
+  --dark-accent-2: #888888;
+  --dark-accent-3: #4E4E4E;
+  --background-dark-transparent: #0000004d;
+  --red: #FF0000;
+}
```

Comment:
CSS does variables now!!! :D 

This makes it easier to "theme" your site. Similar or same colors in certain css rules.